### DISCUSSION TITLE
Trouble uploading to fire beetle

### DISCUSSION THREAD

**Discussion by Wreckingball7** [DCC-EX/EX-T3-WiFi#95] [General]
Sorry for asking such newbie questions.  I am new to platformio and can't figure out how to perform the uploads requested. I need a little more info on what to do.  I don't even know how to pick the folder where the files are.
Here is an error I get while trying to upload.  I am sure it is because it is not looking in the correct folder.
Any help would be appreciated.
![Desktop 7-15-2023 1-06-33 PM-419](https://github.com/DCC-EX/EX-T3-WiFi/assets/67566877/37905ccb-db3a-48f1-88ba-24489a8a3473)

**Comment by alex-code** [comment]
No worries, I'm hoping to remove the need for VSCode soon with a web based upload to make it a bit simpler.

How did you download the code?
The `platformio.ini` file in your pic doesn't match mine.

If you download and extract/unzip the file here https://github.com/DCC-EX/EX-T3-WiFi/archive/refs/heads/master.zip

Then in VSCode you can use the `Open Folder` button.
Select the folder you just extracted then the upload options should work.
